src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -270,12 +270,16 @@ private void fireAuditFinished() {
      * @noinspection ProhibitedExceptionThrown
      */
     private void processFiles(List<File> files) throws CheckstyleException {
+        // -@cs[SingleBreakOrContinue] skip file sooner to avoid extra computation
```

Comment:
sorry, it is not allowed in main code.
Please provide reason of why it is not possible to satisfy or just follow a rule as all other code.
We eat food that we produce.

you can make extra filtering statement before this processing loop in this method, or at method above.

Reply:
We have 2 temporary variables between both continues. The whole point of the fix is to not execute one of the temporary variables in this scenario since it uses disk access. So we can't merge these together.
The only other option, besides leaving this as it is, is to move the code after the first  continue into a separate method. If you would rather have that, what would we name the method? `processFilesEx`?